<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>btn_equal_2</class>
 <widget class="QMainWindow" name="btn_equal_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_div">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 77));
color: rgb(0, 176, 164);
font: 700 28pt &quot;Georgia&quot;;
border-style: solid none none none;
border-color: rgb(0, 176, 164);
border-width: 3px;


</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_text_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 61, 77);
font: 500 40pt &quot;Gill Sans MT Condensed&quot;;
border-style: solid;
color: rgb(22, 217, 178);
border-right-color: rgb(0, 176, 164);
border-left-color: rgb(53, 61, 77);
border-bottom-color: rgb(53, 61, 77);
border-top-color: rgb(53, 61, 77);
border-width: 3px;
</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>331</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
background-color: rgb(53, 61, 77);
border-color: rgb(75, 170, 156);
border-width: 2px;
border-radius: 20px

</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mul">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 77));
color: rgb(0, 176, 164);
font: 700 28pt &quot;Georgia&quot;;
border-style: outset;
border-color: rgb(0, 148, 116);
border-width:0px;</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_per">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 77));
color: rgb(0, 176, 164);
font: 700 18pt &quot;Georgia&quot;;
border-style: solid none none none;
border-color: rgb(0, 176, 164);
border-width: 3px;


</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_neg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 77));
color: rgb(0, 176, 164);
font: 700 28pt &quot;Georgia&quot;;
border-style: solid none none none;
border-color: rgb(0, 176, 164);
border-width: 3px;


</string>
    </property>
    <property name="text">
     <string>±</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ac">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 77));
color: rgb(0, 176, 164);
font: 700 18pt &quot;Georgia&quot;;
border-style: solid none none none;
border-color: rgb(0, 176, 164);
border-width: 3px;


</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sub">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 77));
color: rgb(0, 176, 164);
font: 600 38pt &quot;Georgia&quot;;
border-style: outset;
border-color: rgb(0, 148, 116);
border-width:0px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sum">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 77));
color: rgb(0, 176, 164);
font: 700 28pt &quot;Georgia&quot;;
border-style: outset;
border-color: rgb(0, 148, 116);
border-width:0px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_equal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 176, 164);
color: rgb(255, 255, 255);
font: 700 28pt &quot;Georgia&quot;;
border-style: outset;
border-color: rgb(0, 148, 116);
border-width:0px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 61, 77);
font: 600 18pt &quot;Georgia&quot;;
border-style: none;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 61, 77);
font: 600 18pt &quot;Georgia&quot;;
border-style: none;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 61, 77);
font: 600 18pt &quot;Georgia&quot;;
border-style: none;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 61, 77);
font: 600 18pt &quot;Georgia&quot;;
border-style: none;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 61, 77);
font: 600 18pt &quot;Georgia&quot;;
border-style: none;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 61, 77);
font: 600 18pt &quot;Georgia&quot;;
border-style: none;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 61, 77);
font: 600 20pt &quot;Georgia&quot;;
border-style: none;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 61, 77);
font: 600 20pt &quot;Georgia&quot;;
border-style: none;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 61, 77);
font: 600 18pt &quot;Georgia&quot;;
border-style: none;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 61, 77);
font: 600 18pt &quot;Georgia&quot;;
border-style: none;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(53, 61, 77);
font: 600 18pt &quot;Georgia&quot;;
border-style: none;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 176, 164);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 38));
font: 700 20pt &quot;Georgia&quot;;
border-style: solid none none none;
border-color: rgb(0, 176, 164);
border-width: 3px;


</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 38));
color: rgb(0, 176, 164);
font: 700 20pt &quot;Georgia&quot;;
border-style: none;
border-color: rgb(0, 176, 164);
border-width: 3px;


</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 38));
color: rgb(0, 176, 164);
font: 700 20pt &quot;Georgia&quot;;
border-style: none;
border-color: rgb(0, 176, 164);
border-width: 3px;


</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 38));
color: rgb(0, 176, 164);
font: 700 20pt &quot;Georgia&quot;;
border-style: none;
border-color: rgb(0, 176, 164);
border-width: 3px;


</string>
    </property>
    <property name="text">
     <string>cot</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 38));
color: rgb(0, 176, 164);
font: 700 20pt &quot;Georgia&quot;;
border-style: none;
border-color: rgb(0, 176, 164);
border-width: 3px;


</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sqrt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 148, 116, 38));
color: rgb(0, 176, 164);
font: 700 18pt &quot;Georgia&quot;;
border-style: none;
border-color: rgb(0, 176, 164);
border-width: 3px;
border-radius: 10px;


</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_text_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 61, 77);
border-style: solid;
color: rgb(22, 217, 178);
font: 700 18pt &quot;Georgia&quot;;
border-right-color: rgb(0, 176, 164);
border-left-color: rgb(53, 61, 77);
border-bottom-color: rgb(53, 61, 77);
border-top-color: rgb(53, 61, 77);
border-width: 3px;
</string>
    </property>
   </widget>
   <zorder>listView</zorder>
   <zorder>tb_text_1</zorder>
   <zorder>btn_7</zorder>
   <zorder>btn_8</zorder>
   <zorder>btn_9</zorder>
   <zorder>btn_4</zorder>
   <zorder>btn_5</zorder>
   <zorder>btn_6</zorder>
   <zorder>btn_1</zorder>
   <zorder>btn_2</zorder>
   <zorder>btn_3</zorder>
   <zorder>btn_0</zorder>
   <zorder>btn_dot</zorder>
   <zorder>btn_sin</zorder>
   <zorder>btn_cos</zorder>
   <zorder>btn_tan</zorder>
   <zorder>btn_cot</zorder>
   <zorder>btn_log</zorder>
   <zorder>btn_ac</zorder>
   <zorder>btn_div</zorder>
   <zorder>btn_neg</zorder>
   <zorder>btn_sum</zorder>
   <zorder>btn_equal</zorder>
   <zorder>btn_mul</zorder>
   <zorder>btn_sub</zorder>
   <zorder>btn_per</zorder>
   <zorder>btn_sqrt</zorder>
   <zorder>tb_text_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
